app/serializer_book.py

```diff
@@ -0,0 +1,17 @@
+from abc import ABC, abstractmethod
+
+
+class Serializer(ABC):
+    @abstractmethod
+    def serialize(self, title: str, content: str) -> str:
+        pass
+
+
+class JsonSerializer(Serializer):
+    def serialize(self, title: str, content: str) -> str:
+        return f'{{"title": "{title}", "content": "{content}"}}'
+
+
+class XmlSerializer(Serializer):
+    def serialize(self, title: str, content: str) -> str:
+        return f"<title>{title}</title>\n<content>{content}</content>"

```

Comment:
You changed the initial logic of application, you just had to refactor it without logic changes